helm/bigmon/charts/main/values.yaml

```diff
@@ -48,11 +48,19 @@ httpd:
 
 # database config
 database:
```

Comment:
Better to have
```
panda:
  database:

idds:
  host: ...
  database:
```

instead of
```
database:
  panda:
  idds:

idds:
  host: ...
```

Reply:
I did that initially and I thought it looked a bit ugly and changed it. I can change it back, no problem